mcs/class/corlib/System.Reflection/MonoMethod.cs

```diff
@@ -572,6 +572,12 @@ internal void SerializationInvoke (Object target, SerializationInfo info, Stream
 			Invoke (target, new object[] { info, context });
 		}
        #endregion
+
+		// This is required to support certain kinds of serialization
```

Comment:
Because we don't care anymore, you can implement it correctly using get_core_clr_security_level if you really want